docs/install/Knative-with-Kourier.md

```diff
@@ -0,0 +1,161 @@
+---
+title: "Installing Knative with Kourier"
+linkTitle: "Kourier Ingress and Knative"
+weight: 15
+type: "docs"
+---
+
+[Kourier](https://github.com/3scale/kourier) is an open-source lightweight Knative Ingress based on Envoy. It's been designed for Knative, without requiring any additional custom resource definitions (CRDs).
+
+This guide walks you through the installation of the latest version of Knative
+with Kourier as the ingress.
+
+## Before you Begin
+
+Knative requires a Kubernetes cluster v1.14 or newer, as well as a compatible `kubectl`. This guide assumes that you have already [created a Kubernetes cluster](https://kubernetes.io/docs/setup/) and are using
+bash in a Mac or Linux environment.
+
+## Install Knative
+
+Let's do a core install of Knative Serving with the released yaml templates:
+
+1.  To install Knative, first install the CRDs by running the following `kubectl apply`
+    command. This prevents race conditions during the install, which cause intermittent errors:
+
+        kubectl apply --filename https://github.com/knative/serving/releases/download/{{< version >}}/serving-crds.yaml
+
+2.  To complete the install of Knative and its dependencies, next run the
+    following `kubectl apply` command:
+
+        kubectl apply --filename https://github.com/knative/serving/releases/download/{{< version >}}/serving-core.yaml
+
+3.  Monitor the Knative Serving namespace and wait until all of the pods come up with a
+    `STATUS` of `Running`:
+
+    ```
+    kubectl get pods -w -n knative-serving
+    ```
+
+
+## Install Kourier
+
+Knative default installation uses Istio to handle internal and external traffic. If you are just interested in exposing Knative applications to the external network, a service mesh adds overhead and increases the system complexity. Kourier provides a way to expose your Knative application in a more simple and lightweight way.
+
+You can install Kourier with `kubectl`:
+
+```
+kubectl apply \
+  --filename https://raw.githubusercontent.com/3scale/kourier/master/deploy/kourier-knative.yaml
+```
+
+## Configuring the Knative ingress class
+
+Kourier only exposes ingresses that have the "kourier" ingress class. By default Knative annotates all the ingresses for Istio but you can change that by patching the "config-network" configmap as follows: 
+
+```
+kubectl patch configmap/config-network \
+  -n knative-serving \
+  --type merge \
+  -p '{"data":{"ingress.class":"kourier.ingress.networking.knative.dev"}}'
+```
+
+## Configuring DNS
+
+Installing Kourier will create a Kubernetes Service with type `LoadBalancer`.
+This may take some time to get an IP address assigned, during this process, it
+will appear as `<pending>`.  You must wait for this IP address to be assigned
+before DNS may be set up.
+
+Get this external IP address with:
+
+```
+$ kubectl get svc kourier -n knative-serving
```

Comment:
```suggestion
kubectl get svc kourier -n knative-serving
```

Reply:
For consistency with the rest of the Knative docs, omit the command line prompt.